picard/mbjson.py

```diff
@@ -427,9 +427,9 @@ def artist_to_metadata(node, m):
                 ended = value['ended']
                 if ended and "end" in value:
                     m["enddate"] = value['end']
-        elif key == "begin_area":
+        elif key == "begin_area" or key == "begin-area":
             m["beginarea"] = value['name']
-        elif key == "end_area":
+        elif key == "end_area" or key == "end-area":
```

Comment:
Do we need to keep `begin_area` and `end_area` here? Actually this is a straight bugfix, because the begin and end area currently don't show up in artist searches in Picard since a [search query](https://musicbrainz.org/ws/2/artist/?query=beach+boys&fmt=json) will already return `begin-area` / `end-area`. With this small change it works again in Picard.

I added https://tickets.metabrainz.org/browse/PICARD-1471 to track this for the changelog.

Reply:
The search server outputs them correctly, but musicbrainz-server doesn't, e.g. see https://musicbrainz.org/ws/2/artist/ebfc1398-8d96-47e3-82c3-f782abcdb13d?fmt=json

If Picard doesn't actually make any toplevel artist queries, great! We can remove the old properties. I don't believe they would show up in any release lookups.

Reply:
Yes, currently it is only used for searches, so I think this is fine to check only this case.